running/local-environment.rst

```diff
@@ -0,0 +1,151 @@
+How to setup a local deconst environment
+========================================
+
+Deconst uses homebrew on OSX for dependencies. As such, all of the work in this setup guide is based on a healthy homebrew installation, or installing it fresh.
+
+Update brew database or install homebrew
+----------------------------------------
+
+First, we need to setup (or update) homebrew on your mac. If you *already have homebrew installed*, lets just update your homebrew database\: ::
+
+  $ brew update
+
+If, however, you *don't have homebrew installed*, you can install it with a shell command. You can read more about brew installation on the `homebrew website <http://brew.sh>`_. ::
+
+  $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+Once you've either installed or updated your install, you should be able to run :code:`brew --version` and see the current version of your setup. ::
+
+  $ brew --version
+  0.9.5
+
+Install Docker
+--------------
+
+Deconst development is depending on running within docker containers. We'll use homebrew to install docker and various other docker utilities. ::
+
+  $ brew install docker
+  $ brew install docker-compose
+  $ brew install boot2docker
+  $ boot2docker up
+
+Once those steps are completed, we should be able to verify your docker installation is running correctly. ::
+
+  $ docker version
+
+  Client version: 1.6.0
+  Client API version: 1.18
+  Go version (client): go1.4.2
+  Git commit (client): 4749651
+  OS/Arch (client): darwin/amd64
+  Server version: 1.6.0
+  Server API version: 1.18
+  Go version (server): go1.4.2
+  Git commit (server): 4749651
+  OS/Arch (server): linux/amd64
+
```

Comment:
Here, I had to update docker as well when docker version failed. Brew upgrade docker didn't work in this case, probably because I needed to run brew update first. Here's the fix I used:
boot2docker stop
boot2docker download
boot2docker up

Then running docker version gives me what's above.

I can add this info to a Troubleshooting section if you want, after this merges.